src/main/java/com/epam/ta/reportportal/core/project/impl/DeleteProjectHandlerImpl.java

```diff
@@ -96,6 +102,10 @@ public OperationCompletionRS deleteProjectIndex(String projectName, String usern
 				Predicate.isEqual(false)
 		).verify(ErrorType.FORBIDDEN_OPERATION, "Index can not be removed until index generation proceeds.");
 
+		expect(analyzerServiceClient.hasClients(), Predicate.isEqual(true)).verify(ErrorType.UNABLE_INTERACT_WITH_INTEGRATION,
+				"No Analyzer services up. Please deploy Analyzer service to remove index."
```

Comment:
let's have one message for one logic without asking users to deploy analyzer.
'"There are no analyzer deployed."'